qanary-component-QB-DateOfDeathDBpedia/src/main/java/eu/wdaqua/qanary/component/dateofdeathrealpersons/qb/controller/DefaultController.java

```diff
@@ -0,0 +1,28 @@
+package eu.wdaqua.qanary.component.dateofdeathrealpersons.qb.controller;
+
+import eu.wdaqua.qanary.component.dateofdeathrealpersons.qb.QueryBuilderDateOfDeathDBpedia;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.io.IOException;
+
+@Controller
+public class DefaultController {
```

Comment:
@dschiese Please rename to `QueryBuilderDateOfDeathDBpediaController` to express the connection to `QueryBuilderDateOfDeathDBpedia`.

Reply:
Resolved here: https://github.com/WDAqua/Qanary-question-answering-components/commit/e6d0d608a334a41f53f3bc21fc89b2f85086fe3b